pages/common/peerflix.md

```diff
@@ -1,15 +1,23 @@
 # peerflix
 
-> Stream video or audio based torrents to your favorite player.
+> Stream video- or audio-based torrents to a media player.
 
-- Stream a torrent:
+- Stream the largest media file in a torrent given as a magnet link:
 
-`peerflix "{{magnet:?xt=urn:btih:00112233445566}}"`
+`peerflix "{{magnet:?xt=urn:btih:0123456789abcdef0123456789abcdef01234567}}"`
 
-- Stream a torrent to VLC:
+- Stream the largest file in a torrent, given as a torrent URL, to VLC:
 
-`peerflix "{{torrent url|magnet link}}" --vlc`
+`peerflix "{{http://example.net/music.torrent}}" --vlc`
+
+- Stream the largest file in a torrent to MPlayer, with subtitles:
+
+`peerflix "{{torrent_url|magnet_link}}" --mplayer --subtitles subtitle-file.srt`
 
 - Stream all files from a torrent to Airplay:
 
-`peerflix "{{torrent url|magnet link}}" --all --airplay`
+`peerflix "{{torrent_url|magnet_link}}" --all --airplay`
+
+- List the files in a torrent that are available for streaming:
+
+`peerflix "{{torrent_url|magnet_link}}" --list`
```

Comment:
I think should go further up, since I'd probably want to know what a torrent contains before I decide to stream it.

Reply:
Sounds fair. Where would you suggest?

Reply:
Oh, man! I think this one slipped through the net :P

Maybe at the top or below the first example?

Reply:
At the top sounds good, will do.